modules/xray/shared/src/test/scala/XRaySuite.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2019-2020 by Rob Norris and Contributors
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package natchez
+package xray
+
+import munit.FunSuite
+
+class XRaySuite extends FunSuite {
+
+  val rootId = "1-5759e988-bd862e3fe1be46a994272793"
+  val parentId = "53995c3f42cd8ad8"
+
+  val noParent = "Root=1-5759e988-bd862e3fe1be46a994272793;Sampled=1"
+  val parent =
+    "Root=1-5759e988-bd862e3fe1be46a994272793;Parent=53995c3f42cd8ad8;Sampled=1"
+  val notSampled = "Root=1-5759e988-bd862e3fe1be46a994272793;Sampled=0"
+
+  test("header parsing") {
```

Comment:
Maybe overkill, but since you have both header parsing/encoding we could have a scalacheck prop checking the roundtrip.

Reply:
I thought about it. Came to the conclusion that it would be overkill.

Reply:
FWIW I also think it's a good idea, but IMO it can come in a followup PR if someone wants to do it. Similarly there are places where it might be more clear to use JSON literals instead of constructing JSON objects from key/value pairs, but I think it'd be fine to do that later too.

Reply:
Yes I had the same thought about JSON literals :)